3f/pygmentize/2016-11-05.yaml

```diff
@@ -0,0 +1,9 @@
+title: Remote Code Execution
+link: https://github.com/dedalozzo/pygmentize/issues/1
+cve: ~
+branches:
+    master:
+        time: 2016-12-31 00:00:00
+        versions: ['<=1.1']
```

Comment:
Okay. I wasn't quite sure how to approach an application that has no fix yet, but @cebe was kind enough to help me get the upstream PR taken care of. I'll update this PR to reflect.